ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/LedgerWriter.scala

```diff
@@ -17,18 +17,18 @@ import scala.concurrent.Future
   */
 trait LedgerWriter extends ReportsHealth {
 
+  /**
```

Comment:
What's the point of moving this up here? It's a less important method than `commit`.